pom.xml

```diff
@@ -5,7 +5,7 @@
     <parent>
```

Comment:
maybe you can sort in separate commit, that way it would be easier to follow changes

Reply:
Done.